lib/components/search/SearchFilter.vue

```diff
@@ -40,11 +40,11 @@ const props = defineProps({
   },
 })
 
-const isActive = ref(props.activeFilters.value.includes(props.facetName))
+const isActive = computed(() => props.activeFilters.includes(props.facetName))
 const emit = defineEmits(['toggle'])
 
 watchEffect(() => {
-  isActive.value = props.activeFilters.value.includes(props.facetName)
+  isActive.value = props.activeFilters.includes(props.facetName)
 })
```

Comment:
This block can be removed, because it does essentially what `computed` does.

Reply:
makes even more sense...